DataConnectors/CyberpionSecurityLogs.json

```diff
@@ -0,0 +1,93 @@
+{
+  "id": "CyberpionSecurityLogs",
+  "title": "Cyberpion Security Logs (Preview)",
+  "publisher": "Cyberpion",
+  "descriptionMarkdown": "The Cyberpion Security Logs data connector, ingests logs from the Cyberpion system directly into Sentinel. The connector allows users to visualize their data, create alerts and incidents and improve security investigations.",
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "CyberpionActionItems_CL",
+      "baseQuery": "CyberpionActionItems_CL"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description": "Fetch latest Action Items that are currently open",
+      "query": "let lookbackTime = 14d;\nlet maxTimeGeneratedBucket = toscalar(\n   CyberpionActionItems_CL \n   | where TimeGenerated > ago(lookbackTime)\n   | summarize max(bin(TimeGenerated, 1h))\n   );\nCyberpionActionItems_CL\n | where TimeGenerated > ago(lookbackTime) and is_open_b == true\n | where bin(TimeGenerated, 1h) == maxTimeGeneratedBucket\n "
```

Comment:
Query returns syntax error as **is_open_b** column doe not exist in table.

Reply:
I'm not sure how you are querying, but "is_open_b" is part of the [sample data](https://github.com/cyberpion-yotam/Azure-Sentinel/blob/cyberpion-sl-connector/Sample%20Data/Custom/CyberpionActionItems_CL.json), and is one of the fields of this connector.

Reply:
This column is getting inserted into the **value** column as JSON string because of which it not found. Have added comment on the sample data file with details, please have a look.

Reply:
Try it now (updated sample data)